src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -343,6 +361,67 @@ public BiomeGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 
 		pane5.add("Center", PanelUtils.totalCenterInPanel(spawnproperties));
 
+
+
+		JPanel multiproperties = new JPanel(new GridLayout(8, 3, 0, 2));
+		multiproperties.setOpaque(false);
+
+
+		multiproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/use_pure_multi_noise"),
+				L10N.label("elementgui.biome.use_pure_multi_noise")));
+		multiproperties.add(usePureMultiNoise);
+		multiproperties.add(L10N.label("elementgui.biome.multi_noise_common_empty"));
+
+		usePureMultiNoise.setOpaque(false);
+
+		multiproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/moffset"),
+				L10N.label("elementgui.biome.multi_noise_offset")));
+		multiproperties.add(offset);
+		multiproperties.add(L10N.label("elementgui.biome.multi_noise_common_empty"));
+
+		multiproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/mtemperature"),
+				L10N.label("elementgui.biome.multi_noise_temperature")));
+		multiproperties.add(minTemperature);
+		multiproperties.add(maxTemperature);
+
+
+		multiproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/mhumidity"),
+				L10N.label("elementgui.biome.multi_noise_humidity")));
+		multiproperties.add(minHumidity);
+		multiproperties.add(maxHumidity);
+
+
+		multiproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/merosion"),
+				L10N.label("elementgui.biome.multi_noise_erosion")));
+		multiproperties.add(minErosion);
+		multiproperties.add(maxErosion);
+
+
+		multiproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/mweirdness"),
+				L10N.label("elementgui.biome.multi_noise_weirdness")));
+		multiproperties.add(minWeirdness);
+		multiproperties.add(maxWeirdness);
+
+
+		multiproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/mcontinentalness"),
+				L10N.label("elementgui.biome.multi_noise_continentalness")));
+		multiproperties.add(minContinentalness);
+		multiproperties.add(maxContinentalness);
+
+
+		multiproperties.add(HelpUtils.wrapWithHelpButton(this.withEntry("biome/mdepth"),
+				L10N.label("elementgui.biome.multi_noise_depth")));
+		multiproperties.add(minDepth);
+		multiproperties.add(maxDepth);
+
+
+		pane6.add("Center", PanelUtils.totalCenterInPanel(multiproperties));
+
+		usePureMultiNoise.addActionListener(e -> refreshFieldsMultiNoise());
+		refreshFieldsMultiNoise();

```

Comment:
You call it after, so no need to call it here.

Reply:
What do you mean by after?

Reply:
Two lines after

Reply:
Oh sorry, I didn't noticed that. Fix is coming in next commit